Inclui sugestões de ferramentas livres (#103)
okfn-brasil/embaixadoras: Guia-ODD/Guia-ODD2022.md

### 4. Onde realizar o evento?
Devido à pandemia do coronavírus, recomendamos que os eventos sejam realizados preferencialmente online. Caso você queira realizar um evento presencial, assegure-se de seguir os protocolos de segurança evitando aglomerações em ambientes fechados, certificando-se que o ambiente é bem ventilado, exigindo o uso de máscaras, verificando se os participantes estão em dia com as doses de vacinação aplicadas em sua cidade e, se possível, disponibilizando material de higiene como álcool de concentração 60% a 80%.

> - Ferramentas de streaming (preferencialmente aquelas baseadas em software livre e que [respeitam a privacidade dos usuários](https://switching.software/replace/zoom/), como [Jitsi Meet](https://jitsi.org/), [Big Blue Button](https://bigbluebutton.org/), [Jami](https://jami.net/), [OBS Studio](https://obsproject.com/pt-br), [Matrix](https://matrix.org/) e [PeerTube](https://joinpeertube.org/), em vez de plataformas das big tech como Zoom, StreamYard, Google Meet, YouTube)

Reviewer: Como haviam muitas sugestões, acho que podemos quebrar em tópicos aninhados. Também sugiro mudar um pouco o texto para a quebra funcionar melhor

```suggestion
- Ferramentas de streaming
    - Ferramentas baseadas em software livre e que [respeitam a privacidade dos usuários](https://switching.software/replace/zoom/) são preferencialmente indicadas: [Jitsi Meet](https://jitsi.org/), [Big Blue Button](https://bigbluebutton.org/), [Jami](https://jami.net/), [OBS Studio](https://obsproject.com/pt-br), [Matrix](https://matrix.org/) e [PeerTube](https://joinpeertube.org/)
    - Caso as alternativas anteriores não consigam ser utilizadas também há alternativas fechadas: Zoom, StreamYard, Google Meet, YouTube
```

Reviewer 2: Oi, pessoal!

Vou discordar um pouco da redação dos trechos propostos pois, como vocês sabem, temos construído uma relação mais abrangente com a nossa comunidade em relação a preferências por ferramentas e canais. As sugestões baseadas em software livre são muito bem-vindas, mas não podemos constranger as pessoas que porventura não as conheçam/não se sintam confortáveis para utilizá-las e prefiram seguir com os softwares mais conhecidos e amplamente utilizados (ainda que, infelizmente, sejam proprietários). 

```suggestion
- Ferramentas de streaming
    - Ferramentas baseadas em software livre são preferencialmente indicadas: [Jitsi Meet](https://jitsi.org/), [Big Blue Button](https://bigbluebutton.org/), [Jami](https://jami.net/), [OBS Studio](https://obsproject.com/pt-br), [Matrix](https://matrix.org/) e [PeerTube](https://joinpeertube.org/)
    - Ferramentas de software proprietário: [Zoom](https://zoom.us/), [StreamYard](https://streamyard.com/), [Google Meet](https://meet.google.com/), [YouTube](https://www.youtube.com/)
```

Author: Boa ideia, @giuliocc. Dividindo fica mais fácil a leitura. Porém, nem todas as ferramentas citadas são de streaming. Algumas são de videochamadas, outras de chat em texto, outras são mesmo de streaming. Vou tentar organizar melhor fazendo mais subdivisões.

@ArianeCamilo, esse texto pode ser para muita gente a primeira vez que a pessoa ouve falar em alguma ferramenta, mesmo em relação às proprietárias. Por exemplo, antes deste texto eu mesmo nunca havia ouvido falar em StreamYard. Não é porque a ferramenta é proprietária que ela vai ser mais conhecida ou a pessoa vai se sentir menos "constrangida" a utilizá-la. Entendo que devemos apontar todas as alternativas, mas também explicar as consequências de cada escolha, inclusive no que se refere à liberdade do usuário e coleta de dados.

Vou fazer um novo commit com a proposta.

Author: Atualizado!

- Ferramentas de espaços virtuais (ex.: [WorkAdventure](https://workadventu.re/), Gather) onde as pessoas podem se movimentar e se organizar para conversar em grupos espontâneos de tamanho reduzido
- Locais públicos (ex: universidades, bibliotecas, centros culturais, escolas, sedes de conselhos de bairro)
- Sedes de organizações da sociedade civil
- Empresas de tecnologia (é comum que essas empresas reservem espaços em suas instalações para sediar eventos ao longo do ano, inclusive aos finais de semana)
> Atente-se aos prazos para a reserva do local e verifique a infraestrutura disponível (ex: qualidade da internet, lotação do espaço, transporte público, proximidade de lanchonetes e restaurantes etc.)

> Assim que tiver um local definido, disponibilize um link com mais informações (preferencialmente em ferramentas baseadas em software livre e que [respeitam a privacidade dos usuários](https://switching.software/replace/meetup/), como o [Mobilizon](https://joinmobilizon.org/en/), [Gath.io](https://gath.io/), [Gancio](https://gancio.org/), ou [GetTogether](https://gettogether.community/), em vez de usar um evento no Facebook (só acessa quem tem conta), Eventbrite, Sympla, MeetUp e outras plataformas que têm políticas de privacidade questionáveis) ou mesmo um formulário de inscrição e cadastre o evento no [site oficial do ODD 2022](https://opendataday.org/pt_br/). Dessa forma, mais pessoas encontrarão o evento local com facilidade.

Reviewer: Faz bem mais sentido mover o parágrafo para fora dos tópicos mesmo!

Na mesma linha da sugestão anterior:
```suggestion
Assim que tiver um local definido, disponibilize um link com mais informações ou mesmo um formulário de inscrição e cadastre o evento no [site oficial do ODD 2022](https://opendataday.org/pt_br/). Dessa forma, mais pessoas encontrarão o evento local com facilidade.

- Ferramentas para eventos
    - Ferramentas baseadas em software livre e que [respeitam a privacidade dos usuários](https://switching.software/replace/meetup/) são preferencialmente indicadas: [Mobilizon](https://joinmobilizon.org/en/), [Gath.io](https://gath.io/), [Gancio](https://gancio.org/), ou [GetTogether](https://gettogether.community/)
    - Caso as alternativas anteriores não consigam ser utilizadas também há alternativas fechadas: evento no Facebook (limitação: só acessa quem tem conta), Eventbrite, Sympla, MeetUp
```

Reviewer 2: Seguindo a linha do comentário anterior:

```suggestion
Assim que tiver um local definido, disponibilize um link com mais informações ou mesmo um formulário de inscrição e cadastre o evento no [site oficial do Open Data Day](https://opendataday.org/pt_br/). Dessa forma, mais pessoas encontrarão o evento local com facilidade.
- Ferramentas para eventos
    - Ferramentas baseadas em software livre são preferencialmente indicadas: [Mobilizon](https://joinmobilizon.org/en/), [Gath.io](https://gath.io/), [Gancio](https://gancio.org/), ou [GetTogether](https://gettogether.community/)
    - Ferramentas de software proprietário: evento no Facebook (limitado a usuários com conta na plataforma), [Eventbrite](https://www.eventbrite.com.br/), [Sympla](https://www.sympla.com.br/), [MeetUp](https://www.meetup.com/pt-BR/)
```

Author: Boas sugestões!

Movi a parte que fala sobre políticas de privacidade para um parágrafo em separado, assim fica mais organizado também.

Só para ilustrar como isso pode fazer as pessoas desistirem de participar, no ano passado eu não participei de um evento do Open Data Day somente porque tinha como condição aceitar os termos de serviço e política de privacidade do Discord. Essa plataforma tem [ranking nota E](https://tosdr.org/en/service/536) de privacidade, a mesma nota do Facebook. Até escrevi um [texto](https://herrmann.tech/pt/blog/2021/08/10/por-que-eu-discordo-do-discord-ou-como-escolher-alternativas-com-melhores-termos-de-servico.html) sobre isso.

Assim que tiver um local definido, disponibilize um link com mais informações ou mesmo um formulário de inscrição e cadastre o evento no [site oficial do Open Data Day](https://opendataday.org/pt_br/). Dessa forma, mais pessoas encontrarão o evento local com facilidade.

- Ferramentas para eventos
    - Ferramentas baseadas em software livre são preferencialmente indicadas: [Mobilizon](https://joinmobilizon.org/en/), [Gath.io](https://gath.io/), [Gancio](https://gancio.org/), ou [GetTogether](https://gettogether.community/)
    - Ferramentas de software proprietário: evento no Facebook (limitado a usuários com conta na plataforma), [Eventbrite](https://www.eventbrite.com.br/), [Sympla](https://www.sympla.com.br/), [MeetUp](https://www.meetup.com/pt-BR/)

> Ao escolher uma ferramenta ou plataforma, atente-se se há ou não a necessidade da pessoa criar uma conta e ao quão favoráveis aos usuários são os termos de uso e política de privacidade. Caso se inscrever no evento exija um grande número de passos, fazer cadastro com coleta de muitos dados ou tenha uma política desfavorável, algumas pessoas podem desistir de participar do evento.
> 

> > Consulte sites como [ToS;DR.org](https://tosdr.org/pt_BR/frontpage) e [privacidade.digital](https://www.privacidade.digital/) para ver avaliações de privacidade e resumos das políticas de dados adotadas por cada plataforma.

Reviewer: ```suggestion
> Se não conhece alguma ferramenta e tiver tempo de testar, tome um tempinho para experimentar. Veja quais atendem melhor sua necessidade (gratuidade, recursos disponíveis, etc.) e a do seu público (necessidade de cadastro, privacidade, etc.) e escolha a mais adequada. 
>
> Outra dica: termos de uso de serviços geralmente são maçantes, então sites como [ToS;DR.org](https://tosdr.org/pt_BR/frontpage) e [privacidade.digital](https://www.privacidade.digital/) existem para ver avaliações de privacidade e resumos das políticas de dados adotadas por cada ferramenta utilizada.
```
